app/src/main/java/org/projectbuendia/client/sync/SyncAdapter.java

```diff
@@ -103,14 +69,23 @@
      * Select a phase by setting a boolean value of true for the appropriate key.
      */
     public enum SyncPhase {
-        SYNC_USERS,
-        SYNC_LOCATIONS,
-        SYNC_CHART_ITEMS,
-        SYNC_CONCEPTS,
-        SYNC_PATIENTS,
-        SYNC_OBSERVATIONS,
-        SYNC_ORDERS,
-        SYNC_FORMS
+        SYNC_USERS(R.string.syncing_users, new UsersSyncPhaseController()),
```

Comment:
Thanks!

Under your proposal, how does client code request that only a subset of phases are synced? Would you keep the enum for that, or have a different mechanism?